### PR TITLE
React Capstone-Metrics Webapp

### PR DESCRIPTION
**Pull request by enkog**
In this project, I have created a React and Redux mobile web application to display a list of metrics. 
According to the project requirements, I have;

- Selected an API.
- Built files structure for your React app.
- Prepared routes and navigation in my app.
- Made sure that a user fetches data from the API, can display and filter them.
- Created tests for the application.
- Styled your components to match the design provided.
- Deployed the project and test for final details.
- Recorded a video for the project.
- Created a good README and PR description.

[Live Demo](https://metrics-webapp.herokuapp.com/countries)
[Video Link](https://drive.google.com/file/d/1hwoMFe2K4JU15yOrRTv4OGDdJ9qs3ZYT/view?usp=sharing)

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @enkog 👋,

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Header.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+/* eslint-disable react/prop-types */
```
Comment (mikenath223) [inline src/components/Header.js]: - [x] Kindly remove this eslint disable rule and include prop type definition for the `search` prop.

File: src/redux/countriesSlice.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-param-reassign */
```
Comment (mikenath223) [inline src/redux/countriesSlice.js]: - [x] Kindly remove this eslint disable rule it's best to fix the issue in the code.

File: src/components/Header.js (hunk 1 above)
Comment (enkog) [reply src/components/Header.js]: Thank you @mikenath223 
I have resolved this issue

File: src/redux/countriesSlice.js (hunk 2 above)
Comment (enkog) [reply src/redux/countriesSlice.js]: Thank you @mikenath223
I have resolved this issue

File: src/redux/countriesSlice.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
```
Comment (Dipeshtwis) [inline src/redux/countriesSlice.js]: - [x] Still this issue not resolved, please enable all the linters and try to improve the code to fix the linters.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @enkog, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**

-------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @enkog,

Good job on implementing the suggested changes. Well Done.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**